ssas/service/admin/router.go

```diff
@@ -35,14 +35,14 @@ func Server() *service.Server {
 func routes() *chi.Mux {
 	r := chi.NewRouter()
 	r.Use(service.NewAPILogger(), service.ConnectionClose)
-	r.Post("/group", createGroup)
-	r.Get("/group", listGroups)
-	r.Put("/group/{id}", updateGroup)
-	r.Delete("/group/{id}", deleteGroup)
-	r.Post("/system", createSystem)
-	r.Put("/system/{systemID}/credentials", resetCredentials)
-	r.Get("/system/{systemID}/key", getPublicKey)
-	r.Delete("/system/{systemID}/credentials", deactivateSystemCredentials)
-	r.Delete("/token/{tokenID}", revokeToken)
+	r.With(requireBasicAuth).Post("/group", createGroup)
```

Comment:
Combined with the above suggestion to `r.Use()`, all these `.With()` calls can be removed.